docs/using-the-compiler.rst

```diff
@@ -378,12 +378,11 @@ Input Description
         {
```

Comment:
Can you leave a comment here stating the `modelChecker` object is experimental and subject to changes?